util/steambox

```diff
@@ -51,9 +51,12 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+if [[ -e /dev/nvidiactl ]] ; then
+	DRIDEVS+=(/dev/nvidia*)
```

Comment:
@jcorrado Nothing wrong with your reasoning, sir! :)

I already have your updated code, but wanted to let you you in case you tried my "find" line above -- I forgot to add two very important quotation marks:
```bash
DRIDEVS += ( $(find /dev -name 'nvidia*' -type c -print) )
```
I'm re-posting here just in case anyone sees it, tries it, and wonders why it causes an error.